packages/phoenix-ng/projects/phoenix-ui-components/lib/components/ui-menu/overlay/overlay.component.ts

```diff
@@ -84,10 +84,12 @@ export class OverlayComponent implements AfterViewInit {
 
     if (width > this.MIN_RES_WIDTH && height > this.MIN_RES_HEIGHT) {
       overlayCardElement.style.width = width + 'px';
-      overlayCardElement.style.height = height + 'px';
       if (this.keepAspectRatioFixed) {
         overlayCardElement.style.height = width / this.aspectRatio + 30 + 'px';
       }
+      else {
```

Comment:
The `else` should be on the same line. We have a linting setup which will make the pipeline fail. Can you please check the contribution guide and fix this? (https://github.com/HSF/phoenix/blob/main/CONTRIBUTING.md#code-formatting)

Running `yarn lint:fix` should do the trick.